File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/StrainsCreateStrainsRequestItem.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class StrainsCreateStrainsRequestItem
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("CbdLevel")]
        public double? CbdLevel { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("IndicaPercentage")]
        public int? IndicaPercentage { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Name")]
        public string? Name { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("SativaPercentage")]
        public int? SativaPercentage { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("TestingStatus")]
        public string? TestingStatus { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ThcLevel")]
        public double? ThcLevel { get; set; }
    }
}

File: sdks/thunkmetrc-go/wrapper/models/strains_create_strains_request_item.go
package models

type StrainsCreateStrainsRequestItem struct {
    CbdLevel float64 `json:"CbdLevel,omitempty"`
    IndicaPercentage int `json:"IndicaPercentage,omitempty"`
    Name string `json:"Name,omitempty"`
    SativaPercentage int `json:"SativaPercentage,omitempty"`
    TestingStatus string `json:"TestingStatus,omitempty"`
    ThcLevel float64 `json:"ThcLevel,omitempty"`
}
